sirius-pipeline/sirius_outbound.p4

```diff
@@ -91,18 +92,26 @@ control outbound(inout headers_t hdr,
         eni_to_vni.apply();
 
 #ifdef STATEFUL_P4
-            ConntrackOut.apply(0);
+           ConntrackOut.apply(0);
 #endif /* STATEFUL_P4 */
 
+#ifdef PNA_CONNTRACK
+        ConntrackOut.apply(hdr, meta);
+#endif // PNA_CONNTRACK
+
         /* ACL */
-        if (meta.conntrack_data.allow_out) {
+        if (!meta.conntrack_data.allow_out) {
```

Comment:
is this a bug in current version? @marian-pritsak

Reply:
Per Marian, it is a bug and will review full PR by 2/15